src/algorithm/center-of-mass.hpp

```diff
@@ -221,6 +221,46 @@ namespace pinocchio
                        DataTpl<Scalar,Options,JointCollectionTpl> & data,
                        const bool computeSubtreeComs = true);
 
+  ///
+  /// \brief Computes the jacobian of the center of mass of the given subtree according to a particular joint configuration.
+  ///        In addition, the algorithm also computes the Jacobian of all the joints (\sa pinocchio::computeJointJacobians).
+  ///
+  /// \tparam JointCollection Collection of Joint types.
+  /// \tparam ConfigVectorType Type of the joint configuration vector.
+  ///
+  /// \param[in] model The model structure of the rigid body system.
+  /// \param[in] data The data structure of the rigid body system.
+  /// \param[in] q The joint configuration vector (dim model.nq).
+  /// \param[in] parentJointIdIn Index of the parent joint supporting the subtree.
+  /// \param[out] J A reference on the Jacobian matrix where the results will be stored in (dim 3 x model.nv). You must fill J with zero elements, e.g. J.fill(0.).
+  ///
+  template<typename Scalar, int Options, template<typename,int> class JointCollectionTpl, typename ConfigVectorType>
+  inline void
+  jacobianSubtreeCenterOfMass(const ModelTpl<Scalar,Options,JointCollectionTpl> & model,
+                              DataTpl<Scalar,Options,JointCollectionTpl> & data,
+                              const Eigen::MatrixBase<ConfigVectorType> & q,
+                              const JointIndex & parentJointIdIn,
+                              typename DataTpl<Scalar,Options,JointCollectionTpl>::Matrix3x & JacobianOut);
+
+  ///
+  /// \brief Computes the jacobian of the center of mass of the given subtree according to the current value stored in data.
+  ///        It assumes that forwardKinematics has been called first.
+  ///
+  /// \tparam JointCollection Collection of Joint types.
+  /// \tparam ConfigVectorType Type of the joint configuration vector.
+  ///
+  /// \param[in] model The model structure of the rigid body system.
+  /// \param[in] data The data structure of the rigid body system.
+  /// \param[in] parentJointIdIn Index of the parent joint supporting the subtree.
```

Comment:
I would suggest using a better naming: replacing `parentJointIdIn` with `root_subtree_id`

Reply:
This is just a suggestion. Any other idea is welcome.

Reply:
I though you were also using upperCamelCase for variable names when applicable ?

Reply:
My suggestion is much more related to the naming choice.